### ISSUE TITLE
Update env vars before being set in config

### ISSUE THREAD

**Issue by gilbitron** [TappNetwork/laravel-aws-secrets-manager#27]
Manually maintaining the list of config variables that need to be overridden in `variables-config` is cumbersome and a headache for larger projects. Would there be a way to inject the environment variables before the config is set? Maybe using a command to pull and set the variables in the environment before running `php artisan config:cache`? So you could do something like this:

```shell
php artisan fetch-secrets-from-aws
php artisan config:cache
```

**Comment by devsi** [comment]
I agree with this too. On larger projects, often config files like `services.php` or similar will contain accessors via `config()` en-masse. Maintaining a second list of all of these as dot notated accessors for the specific array is a real nightmare.

**Comment by nikita-a2zweb** [comment]
Ditching this package because of this issue. Good package otherwise
